proposals/stage-0/TRAVEL_FUND/README.md

```diff
@@ -0,0 +1,56 @@
+
+We shall review the process in this proposal, but the intent is to leave the mechanics of the fund unchanged, simply moving from the purview of the Node.js project into the domain of the OpenJS Foundation, overseen by the CPC. This operation can be viewed as a refactor which moves the travel fund up one level in the organizational tree.
+
+The current fund is used mostly for attending collaborator summits, and we do not expect this to change.
+
+## Difference from the current process
+
+The existing process requires approval from two members of the Node.js TSC and two members of the Node.js Community Committee.
+The new process will require approval from four members of the CPC.
+
+The role of fund "treasurer" will be removed, as it has been found to be redundant and is not currently filled (see https://github.com/openjs-foundation/cross-project-council/pull/268#issuecomment-513888283).
+
+## Practical Specifics
+
+- Move relevant files from nodejs/admin to openjs-foundation/cross-project-council such as:
+  - https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md
+  - https://github.com/nodejs/admin/tree/master/TravelFunds
+  - https://github.com/nodejs/admin/blob/master/travel-visas.md
+  - https://github.com/nodejs/admin/blob/master/expense-report-template.xls
+  - https://github.com/nodejs/admin/blob/master/reimbursement_process.pdf
+- Modify these files to account for OpenJS rather than Node.js organizations
+- Remove mentions from nodejs/admin README, add links to CPC README
+- Remove the role of treasurer
+
+## Required Resources
+
+@brianwarner reports that everything can stay the same for the tools and accounts involved.
+No other resources required.
+
+## How would success be measured?
+
+Success will be achieved if Node.js and non-Node.js members use the travel fund to attend a Foundation-related event, and nobody notices the difference.
+
+## Why this proposal is important
+
+Organizationally, the Node.js project is at the same level as the other impact projects in the Foundation, and so the travel fund should benefit all projects equally in opportunity. Plus, the sponsorships which pay for the fund sponsor the entire Foundation, not just the Node.js project.
+
+## Unresolved Questions
+
+- Who is the current treasurer of the fund?
+
+## What is necessary to complete this proposal
+
+- Definition and wording of the travel fund mechanics, mostly moving files and updating references.
```

Comment:
I would like approval and buy in from both node.js TSC and CommComm on this one before moving it to Stage 3.

Reply:
+1

Reply:
I was hoping for a couple suggested people I could @mention here, to get feedback from the get-go. Who would you suggest?

Reply:
It's unfortunate that @-mentioning nodejs teams doesn't cause notifications in this org. Otherwise, it would be a simple @nodejs/tsc @nodejs/community-committee

Reply:
Oh okay, that’s easy. I’ll create an issue in nodejs/admin and mention both those teams. Thanks.

Reply:
We decided in today’s meeting that an issue should be created in the admin repo, @mention the two teams, include a link to this proposal, and additionally include a link to the proposal staging process.
But we thought to merge this as a stage-0 proposal first, after the conversations here are resolved, so the conversation about promotion to stage-1 can start fresh with larger involvement.